src/qml/WatchfaceSelector.qml

```diff
@@ -108,7 +109,7 @@ Item {
 
                         z: 2
                         visible: !previewPng.previewExists
-                        active: visible
+                        active: visible || (watchface === folderModel.folder + "/" + fileName)
```

Comment:
This `watchface === folderModel.folder + "/" + fileName` is used quite extensively, maybe we should extract it into a bool property selected so it's easier to understand what it means.